.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
**nitpick:** It looks like prettier is changing a lot of `'` to `"`.  Would it be possible to just have it default to single quotes?

Reply:
Let me find where that's set and I'll do that

Reply:
It looks like we need to choose one or the other, and either direction results in a lot of changes. I tend to like the appearance of double-quotes more